declarations/Uber.json

```
@@ -17,6 +17,11 @@
         "p > img"
       ],
       "executeClientScripts": true
+    },
+    "Trackers Policy": {
+      "fetch": "https://www.uber.com/legal/fr/document/?name=cookie-notice&country=france&lang=fr",
+      "executeClientScripts": true,
+      "select": ".ka"
```

In this case, i believe that selecting by `#legal-info` could me more maintenable than selection by a by non-sense css class name `.ka`

```suggestion
      "select": [
        "#legal-info"
      ]
```